website/src/content/blog/2025-03-31-typespec-1-0-release/typespec_1_0_release.md

```diff
@@ -0,0 +1,215 @@
+authorAvatar: assets/img/authors/mario_guerra.png
+tags:
+  - release
+  - announcement
+---
+
+We're excited to announce the TypeSpec 1.0 Release Candidate, marking a significant milestone for the TypeSpec project. TypeSpec embodies a simple yet powerful principle: write your API once, and let TypeSpec take care of the rest. This straightforward approach allows you to focus on designing the perfect API contract while TypeSpec handles generating all the artifacts you need - from OpenAPI specifications to client libraries and server code scaffolding - saving you time and eliminating inconsistencies.
+
+## What is TypeSpec?
+
+TypeSpec is a language for describing API contracts. Using TypeSpec, you write your API definition in a concise, human-readable format, and TypeSpec takes care of generating:
+
+- API specifications (OpenAPI, JSON Schema, Protocol Buffers)
+- Server-side code skeletons (C#, JavaScript)
+- Client libraries in multiple languages (JavaScript, Python, C#, Java)
+- Documentation
+- Custom formats through our emitter framework
+
+This API-first development approach accelerates your initial API development by generating boilerplate code automatically, while also eliminating the need to maintain multiple, separate artifacts that can easily drift out of sync as your API evolves.
+
+## Who's using TypeSpec?
+
+TypeSpec is already being used by teams both inside and outside Microsoft to streamline their API development workflows:
+
+> "One of the highlights [of using TypeSpec] was our ability to author API specs in a very simple and intuitive language. This meant that even [our PMs] who are not engineers were able to understand these API specifications. They could review them, make comments, and participate in designing those APIs."
+>
+> — _MS Learn Team_
+
+> Before the introduction of TypeSpec and our move to an API first approach, an OAS [OpenAPI Specification] was a byproduct of the implementation phase. Manual OAS reviews were happening at the end of the development cycle and changes were contentious because of the code rework implied and the likely impact to the release schedule. TypeSpec provides a way to embed [our] guidelines and rules directly in the design. Reviews are now focused more on the design than on the rules and requires less time." [(read more)](../2024-11-04-typespec-at-lseg/blog.md)
+>
+> — _London Stock Exchange Group_
+
+You can explore more real-world TypeSpec usage on our [videos page](https://typespec.io/videos/), featuring presentations from both Microsoft teams and external community members showing how they're using TypeSpec to improve their API development workflows.
+
+## What's in the 1.0 Release Candidate
+
+The 1.0-RC includes a foundation of stable core components alongside preview features that are still actively being developed based on community feedback.
+
+### Stable components
+
+These components have been thoroughly tested and are considered ready for testing in your production environment:
```

Comment:
ready for testing or ready for use?

Reply:
ready for testing, we aren't ready to make claims about readiness for production till we get some real world feedback. We'll reserve "ready for production" for the actual 1.0.